.circleci/config.yml

```diff
@@ -231,6 +231,9 @@ jobs:
           tag: latest
           use_tarfile: true
       - release-management/create-github-release
+    environment:
```

Comment:
This is based on the instructions found here. 

    https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-job
    https://circleci.com/docs/2.0/configuration-reference/#environment

I believe the indentation is correct, anyone reviewing please confirm.

Reply:
The indentation looks right to me. Does it matter that this is below the `steps` section? Probably not?

Reply:
I don't believe order matters. the examples didn't have a `steps` node, but the `environment` was listed last. the unit tests run without an error, so it should at least match the schema.